src/main/external-resources/renderers/Roku-TV8.conf

```diff
@@ -0,0 +1,62 @@
+#----------------------------------------------------------------------------
+# Profile for Roku TV running Network Streaming Player 8.
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Roku TV (NSP 8)
+RendererIcon = roku-roku3.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Roku/DVP-8.0 (308.00E04156A)
+# ============================================================================
+#
+
+UserAgentSearch = Roku/DVP-8.
+
+LoadingPriority = 1
+
+TranscodeVideo = MPEGTS-H265-AAC
+TranscodeAudio = MP3
+MaxVideoBitrateMbps = 24
+CustomFFmpegOptions = -x264opts cabac=1 -ac 2
+CustomMencoderOptions = -channels 2
+HalveBitrate = true
+MediaInfo = true
+
+# Supported video formats:
+Supported = f:3gp|3g2         v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac|aac-he|ac3|eac3|lpcm|mp3|wma   m:video/3gpp
```

Comment:
```aac``` isn't a valid codec, you probably want ```aac-lc```: https://github.com/UniversalMediaServer/UniversalMediaServer/blob/a83d67c77df019c316eb253511202c2ffaf6ad03/src/main/external-resources/renderers/DefaultRenderer.conf#L569
```aac-he``` isn't valid either, it should be ```he-aac```: https://github.com/UniversalMediaServer/UniversalMediaServer/blob/a83d67c77df019c316eb253511202c2ffaf6ad03/src/main/external-resources/renderers/DefaultRenderer.conf#L600